test/mpfr.jl

```diff
@@ -475,7 +475,7 @@ end
     @test precision(z) == 240
     x = BigFloat(12)
     @test precision(x) == old_precision
-    @test_throws DomainError setprecision(1)
+    @test_throws DomainError setprecision(0)
```

Comment:
Probably we should explicitly test that `setprecision(1)` is allowed and results in 1-bit BigFloats.

Reply:
Sorry I did not get 😔😔

Reply:
```suggestion
    @test precision(setprecision(1) do; BigFloat(23); end) == 1  # minimum-precision
    @test_throws DomainError setprecision(0)
```